source/includes/_embed2.md.erb

```diff
@@ -381,6 +383,25 @@ This action requires the `developer_proactive` feature flag to be turned on.
 
 Deletes the `chatter` used to fetch conversation logs for an end-user from storage. When a user opens a new Chat window a new `chatter` will be generated.
 
+#### evaluateCampaignConditions
+
+```js
+window.adaEmbed.evaluateCampaignConditions({
+  ignoreStatus: false,
+  ignoreFrequency: false
+});
+```
+
+`evaluateCampaignConditions(options: object): Promise<void>;`
+
+This is similar to `triggerCampaign` - however, instead of triggering a specific Campaign, `evaluateCampaignConditions` evaluates the trigger conditions of the Campaigns in priority order and triggers the first Campaign (if any) whose conditions are matched.
```

Comment:
```suggestion
This is similar to `triggerCampaign` - however, instead of triggering a specific Campaign, `evaluateCampaignConditions` evaluates the trigger conditions of the Campaigns in priority order and triggers the first Campaign (if any) whose conditions are matched. This can be useful if e.g. the Embed application cannot determine that the route has changed and the campaign trigger rules need to be evaluated again.
```